### DISCUSSION TITLE
Getting hi and low byte of a label

### DISCUSSION THREAD

**Discussion by cgimenez** [vhelin/wla-dx#622] [General]
Hi all,

Given
```
v8p:
.DB 2, 0
```

What is the right syntax to get the hi and low bytes of this label address ?

Is it

```
LDA #>v8p
LDA #<v8p
```
or
```
LDA >v8p
LDA <v8p
```

**Comment by vhelin** [comment]
Here are some pointers about this issue:

https://wla-dx.readthedocs.io/en/latest/arithmetics.html
https://wla-dx.readthedocs.io/en/latest/functions.html

![image](https://github.com/vhelin/wla-dx/assets/6417746/0b49b508-2e34-4067-8b6c-f0c712b1ec11)

![image](https://github.com/vhelin/wla-dx/assets/6417746/01829974-20e2-49a1-a8d9-607a48739645)

If you are talking about 6502 then

LDA #>v8p

and

LDA >v8p

are different instructions. I'm guessing (I'm no 6502 programmer) that you want "LDA #>v8p"... I could be wrong, though. :)

**Reply by cgimenez** [reply]
Thanks for your reply (and yes I'm on the 6502 side, forgot to mention)

Yes I also guess that I want 
`LDA #>v8p`

If v8p resolve to $2000
I want to LDA with #20

But what does 
`LDA >v8p`
?

Have not coded for 8bit cpus for a very very very long time, so forgive me if I mismatch things ;-)

**Reply by vhelin** [reply]
I this here's the answer: https://wiki.cdot.senecacollege.ca/wiki/6502_Addressing_Modes

![image](https://github.com/vhelin/wla-dx/assets/6417746/fb7c41fc-7083-4118-b672-b16d8da416ad)

**Reply by vhelin** [reply]
So "LDA >v8p" -> "LDA $20" -> load accumulator from zero page address $20
